docs/sphinx/manuals/roc_recv.rst

```diff
@@ -38,6 +38,7 @@ Options
 -1, --oneshot             Exit when last connected client disconnects (default=off)
 --poisoning               Enable uninitialized memory poisoning (default=off)
 --beeping                 Enable beeping on packet loss  (default=off)
+--miface=IFACE_IP         IP address of the network interface on which to join to the multicast group
```

Comment:
It's better to keep the same order of options in .rst and .ggo (and --help).

You can use this command to generate rst table:

    roc-recv --help | ./scripts/help2rst.py